src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/Inputs/OneRunWithAllReportingDescriptors.sarif

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-1.0.0",
+  "version": "1.0.0",
+  "runs": [
+    {
+      "rules": {
+        "C2003": {
+          "id": "C2003",
+          "name": "Rule C2003",
+          "shortDescription": "Rules were meant to be broken.",
+          "fullDescription": "This is a test for validating modification of tool.driver.ruleDescriptors",
+          "configuration": "disabled",
+          "defaultLevel": "pass"
```

Comment:
>"pass" [](start = 26, length = 6)

a default level of pass? this is wrong, it s/be a failure level. did you get this pattern from an existing test?

Reply:
yes, let me fix it in source as well!

---
In reply to: [260550146](https://github.com/Microsoft/sarif-sdk/pull/1309#discussion_r260550146) [](ancestors = 260550146)